isis/src/base/objs/SurfacePoint/SurfacePoint.cpp

```diff
@@ -279,6 +256,8 @@ namespace Isis {
     else {
       p_z = new Displacement(z);
     }
+    
+    p_localRadius = NULL;
```

Comment:
This should check if there is an existing local radius and delete it before assigning to NULL. Current implementation is a potential memory leak

Reply:
Fixed